Common/include/option_structure.hpp

```diff
@@ -1265,10 +1291,25 @@ inline LM_ParsedOptions ParseLMOptions(const LM_OPTIONS *LM_Options, unsigned sh
     LMParsedOptions.Correlation = TURB_TRANS_CORRELATION::MENTER_LANGTRY;
     NFoundCorrelations++;
   }
+  if (IsPresent(LM_OPTIONS::MENTER_SLM)) {
+    LMParsedOptions.Correlation_SLM = TURB_TRANS_CORRELATION_SLM::MENTER_SLM;
+    NFoundCorrelations_SLM++;
+  }
+  if (IsPresent(LM_OPTIONS::CODER_SLM)) {
+    LMParsedOptions.Correlation_SLM = TURB_TRANS_CORRELATION_SLM::CODER_SLM;
+    NFoundCorrelations_SLM++;
+  }
+  if (IsPresent(LM_OPTIONS::MOD_EPPLER_SLM)) {
+    LMParsedOptions.Correlation_SLM = TURB_TRANS_CORRELATION_SLM::MOD_EPPLER_SLM;
+    NFoundCorrelations_SLM++;
+  }
 
   if (NFoundCorrelations > 1) {
     SU2_MPI::Error("Two correlations selected for LM_OPTIONS. Please choose only one.", CURRENT_FUNCTION);
   }
+  if (NFoundCorrelations_SLM > 1) {
+    SU2_MPI::Error("Two correlations selected for Simplified model into LM_OPTIONS. Please choose only one.", CURRENT_FUNCTION);
```

Comment:
```suggestion
    SU2_MPI::Error("Two correlations selected for simplified LM_OPTIONS. Please choose only one.", CURRENT_FUNCTION);
```

Reply:
I'd stick with my change since the options are for the simplified model. They are not simplified options.